Issue #49, Link conformance penalty decision to Section 3.9 diagnosis… (#54)
ietf-wg-scone/appman: draft-ietf-scone-applicability-manageability.md

To simplify the measurement function, reduce computational load, or offload this
function to another node in the network, operators can select any value larger

> than the baseline 67-second window ({{Section 5.2 of I-D.ietf-scone-protocol}})

Reviewer: ```suggestion
than the baseline 67 second window ({{Section 5.2 of I-D.ietf-scone-protocol}})
```

for their measurement and averaging period.

Because some applications will not support SCONE, and others either will not or cannot follow

> the provided throughput advice, operators have flexibility in how they handle violations.

Reviewer: ```suggestion
the provided throughput advice, operators have flexibility in how they handle flows that exceed the limits that are set in their policies.
```

> Before applying penalties, operators can use the diagnostic approach described in

Reviewer: ```suggestion
Before applying rate control mechanisms, operators can use the diagnostic approach described in
```

{{monitoring-and-logging}} to confirm that a sustained violation reflects application

> non-compliance rather than a delivery failure per {{Section 7.3 of I-D.ietf-scone-protocol}}.

Reviewer: ```suggestion
{{monitoring-and-logging}} to confirm that the flow requires intervention
in order to maintain target rates per {{Section 7.3 of I-D.ietf-scone-protocol}}.
```

> If the monitoring function detects a violation where an application is not respecting the

Reviewer: ```suggestion
If the monitoring function detects that an application is not respecting the
```

signaled throughput advice, the network can employ a throttling fallback. This involves falling

> back to traditional rate-limiting mechanisms, such as dropping or delaying packets, to ensure

Reviewer: ```suggestion
signaled throughput advice, the network can employ traditional rate-limiting mechanisms, such as dropping or delaying packets, to ensure
```
